src/control/server/storage/faults.go

```diff
@@ -30,7 +32,9 @@ func FaultScmNotHealthy(sockID uint) *fault.Fault {
 	return storageFault(
 		code.ScmBadRegion,
 		fmt.Sprintf("PMem region on socket %d is unhealthy", sockID),
-		fmt.Sprintf("Refer to the ipmctl instructions in the troubleshooting section of the DAOS admin guide to check PMem module health and replace faulty PMem modules. %s", recreateRegionsStr))
+		fmt.Sprintf("Refer to the ipmctl instructions in the troubleshooting section of "+
+			"the DAOS admin guide to check PMem module health and replace faulty PMem "+
+			"modules. %s", recreateRegionsStr))
```

Comment:
Are you making these changes based on personal preference or a linter rule or what? We keep going back and forth on line lengths and these kinds of changes add noise to an already lengthy review, because reviewers have to determine if the content has changed or if it's just someone deciding arbitrarily that they don't like how long the line is.

Reply:
I don't like Long lines as I find it harder to read, I will remove non-critical line break changes.